### ISSUE TITLE
When some rows have dropdown other rows are not editable

### ISSUE THREAD

**Issue by Bastituta** [plotly/dash-table#836]
If I use dropdowns in a dash table with Editable = true, I cannot edit values in the other rows. This has been reported my several people, but I have not found a fix yet.
Some people say that it used to work, but not any longer.

**Comment by Bastituta** [comment]
> If I use dropdowns in a dash table with Editable = True, I cannot edit values in the other rows. This has been reported by several people, but I have not found a fix yet.
> Some people say that it used to work, but not any longer. Any ideas?

Let me rephrase the issue. When you have an editable table and in a certain column two different type of rows. Some rows have dropdown values and therefore editable, whereas others have a single value. The single values are not editable.
This means that you need to split the table in a table with only rows with drop down values and a table with only single values, in order to make them editable. 
Is this  expected behaviour? It would be nice if you can mix single values in some rows with drop down values in other rows and have all rows editable.

**Comment by aGitForEveryone** [comment]
I would like to bump this thread. This is still an issue. I have a column where some cells are dropdown fields, and other cells are numeric data or strings. However, I cannot alter the cells that contain numeric or string data. I can only change the cells with dropdowns.

**Comment by jeetoza92** [comment]
I am having a same issue. Is there any fix for this?

**Comment by AaronWoodrow** [comment]
I'm having the same issue here.
e.g.  `columns=([{'id': 'config_value', 'name': 'config_value', 'presentation': 'dropdown', 'editable': True}])`
I also have `editable=True` as a property of the DataTable

**Comment by thcha67** [comment]
Same issue here still.

**Comment by alexrblohm** [comment]
I also have this issue. I would like to use dropdowns in some columns and then freely edit other columns
